### DISCUSSION TITLE
Take into account external probe with Inkbird TH1

### DISCUSSION THREAD

**Discussion by 1technophile** [theengs/app#42] [General]
**Describe the bug**
Currently, we support the IBS-TH1 with its internal sensor. This sensor can also be connected to an external probe, when done the decoder advertise the data like below:
```
{"id":"","mac_type":0,"name":"sps","rssi":-64,"brand":"Inkbird","model":"TH Sensor","model_id":"IBS-TH1","cidc":false,"tempc2_ext":25.17,"tempf2_ext":77.306,"hum":60.31,"batt":81}
```

Note that it also advertize the internal sensor data:
![image](https://user-images.githubusercontent.com/12672732/200080110-beb3c76a-5c93-43be-ad6b-1ee46e3b8e21.png)

**Expected behavior**
To be discussed

**Environment (please complete the following information):**
* Tested with this [build](https://github.com/theengs/app/actions/runs/3372530615) on windows

**Comment by 1technophile** [comment]
We could:
1) consider the internal sensor temperature and humidity when no probe is connected in the device display and charts, and the probe humidity and temperature in the main display and charts when it is connected
2) display all the information like a Generic sensor without a chart
3) have a chart for the sensor and the probe, maybe the more expensive
4) differentiate the device when the probe is connected (will appear as 2 device in Theengs App)

**Comment by 1technophile** [comment]
Also @DigiH it seems that we have humidity with the external probe, as I see 2 different humidity values on the screenshot

**Reply by DigiH** [reply]
> Also @DigiH it seems that we have humidity with the external probe, as I see 2 different humidity values on the screenshot

I would not even think this possible, with the external probes being totally encased in either metal or plastic/silicon, therefore not allowing any airflow over any humidity sensor, if it was inside the external probes.

**Reply by DigiH** [reply]
The humidity recorder when an external probe is connected is the humidty of the internal humidity sensor, as also stated in the manual

> Note: The built-in sensor only reads the humidity when the external temperature probe is inserted.

That same note also makes me wonder how you were seeing both external and internal temperatures, unless it was always the external temperature, and when implementing it into the decoder the external probe property being indicated by 

`"condition":["manufacturerdata", 9, "!", "0"],`

is not the only condition, but was the only condition I found with my TH1s when testing the implementation.

Even though I wished that both temperatures were transmitted, as that would be even nicer ;)

**Reply by 1technophile** [reply]
On the screenshot you can see 2 different temperatures transmitted at the same time. I will conduct some tests tomorrow.

**Reply by 1technophile** [reply]
Seems that I can't reproduce the behavior of having both temperatures:
![image](https://user-images.githubusercontent.com/12672732/200117459-9a07428d-f592-4db6-b6ff-473d616dd791.png)
I will let it run for some time and see
